### PR TITLE
[browser] disable enablePerfMeasure by default

### PR DESCRIPTION
**Pull request by pavelsavara**
Fix perf regression from https://github.com/dotnet/runtime/pull/85730

### CODE REVIEW

**Review by kg — APPROVED** [review]

File: src/mono/wasm/runtime/loader/globals.ts [hunk 1]
```diff
@@ -39,7 +51,10 @@ export function setLoaderGlobals(
         config: { environmentVariables: {} }
     });
     Object.assign(runtimeHelpers, {
+        mono_wasm_bindings_is_ready: false,
+        javaScriptExports: {} as any,
         config: globalObjects.module.config,
+        enablePerfMeasure: true,
```
Comment (lewing) [inline src/mono/wasm/runtime/loader/globals.ts:57]: did you mean to enable this here?

Comment (pavelsavara) [reply src/mono/wasm/runtime/loader/globals.ts]: It almost doesn't matter because the normalizeConfig will replace it. But yeah, removed.

**Comment by pavelsavara** [comment]
Somehow this doesn't trigger right CI lanes!

File: src/mono/wasm/runtime/loader/globals.ts (hunk 1 above)
Comment (kg) [reply src/mono/wasm/runtime/loader/globals.ts]: It gets automatically disabled later during startup, as I understand it

**Pull request merged**